skyportal/tests/api/test_instrument.py

```diff
@@ -31,6 +36,9 @@ def test_token_user_post_get_instrument(super_admin_token):
             'band': 'NIR',
             'filters': ['f110w'],
             'telescope_id': telescope_id,
+            'field_data': pd.read_csv(datafile).to_dict(orient='list'),
```

Comment:
We need to test that the fields/tiles that are expected to be generated actually are

Reply:
Do you have a suggestion how to do so aside from the counting?

Reply:
Wait a reasonable amount of time, then fetch fields/tiles and make sure the ones that should be there are there

Reply:
Ok I sliced it to only add a few tiles. Lemme know what you think.